go.mod

```diff
@@ -174,6 +174,11 @@ require (
 	github.com/mailgun/minheap v0.0.0-20170619185613-3dbe6c6bf55f // indirect
 )
 
+// Do not remove. It breaks github.com/gravitational/go-oidc.
```

Comment:
```suggestion
// Do not remove. It breaks github.com/gravitational/go-oidc.
// TODO(tobiaszheller): Declare dependencies in the fork and remove this.
```

Reply:
i will remember without note, just don't want to waste green light :)